index.html

```diff
@@ -0,0 +1,132 @@
+    <header class="mainnav">
+      <nav class="navt">
+        <!--Top navigation  items to the Left-->
+        <div class="navL">
+          <div class="Tym">9:41</div>
+          <div class="nameL">John Doe</div>
+        </div>
+
+        <!--Top navigation  items to the Left-->
+        <div class="navR">
+          <div class="navv">
+            <a><i class="fa fa-signal"></i></a>
+            <a><i class="fa fa-wifi"></i></a>
+            <a><i class="fa fa-battery-full"></i></a>
+          </div>
+          <!--Hamburge Menu-->
+          <div class="HamB-menu">
+            <div class="Nvr hr1"></div>
+            <div class="Nvr hr2"></div>
+            <div class="Nvr hr3"></div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!--Main Flexbox Container-->
+    <div class="container flame">
+      <!--Headed Details-->
+      <div class="Mname flame"><h1>Laura Jong</h1></div>
+      <div class="SE flame"><h3>Software Developer</h3></div>
+      <!--Introduction-->
+      <div>
+        <p class="detail">
+          I can build a Product, Feature or Website. Look through some of my
+          work and experience! if you like what you See and have a project you
+          need coded, dont hesitate to contact me.
+        </p>
+      </div>
+      <div class="HLR"></div>
+      <!--Lists -->
+      <div class="tbl">
+        <ul>
+          <li class="HL">Language</li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+        </ul>
+
+        <ul>
+          <li class="HL">Frameworks</li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+
+        <ul>
+          <li class="HL">Skills</li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+      <!--Eclipse Elements-->
+      <div class="eclips"></div>
+      <div class="eclips"></div>
+      <div class="eclip1"></div>
+      <div class="eclips1"></div>
+
+      <div class="HLR"></div>
+      <!--Social Media Navigators-->
+
+      <div class="social">
+        <a href="https://github.com/">
+          <i class="fa fa-github fa-2x" aria-hidden="true"></i
+        ></a>
+        <a href="https://www.facebook.com/">
+          <i class="fa fa-facebook fa-2x" aria-hidden="true"></i>
+        </a>
+        <a href="https://www.linkedin.com/">
+          <i class="fa fa-linkedin-in fa-2x" aria-hidden="true"></i>
+        </a>
+        <a href="https://www.angellist.com">
+          <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
+        </a>
+        <a href="https://twitter.com/i/flow/login">
+          <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
+        </a>
+        <a href="https://www.instagram.com">
+          <i class="fa fa-instagram fa-2x" aria-hidden="true"></i>
+        </a>
```

Comment:
- [ ] As mentioned by the previous reviewer the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)
e.g:
```
<ul class="social">
   <li> 
      <a href="https://github.com/">
         <i class="fa fa-github fa-3x" aria-hidden="true"></i>
      </a>
   </li>
   ....
</ul>
```